sentry/connection_config.go

```diff
@@ -59,6 +72,9 @@ func getClient(ctx context.Context, d *plugin.QueryData) (*sentry.Client, error)
 		httpClient := oauth2.NewClient(ctx, tokenSrc)
 
 		client := sentry.NewClient(httpClient)
+		if baseUrl != "" {
```

Comment:
Looking at `NewOnPremiseClient`'s code (https://github.com/jianyuan/go-sentry/blob/main/sentry/sentry.go#L109), I wonder if it's better to:
- Create the var `baseUrl` with a default value of `"https://sentry.io/"` (based off of https://github.com/jianyuan/go-sentry/blob/main/sentry/sentry.go#L23)
- Check for env var and config arg are set (like you are today)
- When creating the client, always use `client, err = sentry.NewOnPremiseClient(baseUrl, httpClient)`, since this is what the SDK uses by default anyway with `NewClient`.
  - We can also check if `err` is not nil and then return:
    ```
    plugin.Logger(ctx).Error("getClient", "client_error", err)
    return nil, err
    ```
    
@beudbeud Thoughts on this change?